aries-site/src/themes/aries.js

```diff
@@ -3,6 +3,12 @@ import { deepMerge, normalizeColor } from 'grommet/utils';
 
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
+  // To be removed after NEXT layer overlay PR is accepted on NEXT branch
+  layer: {
```

Comment:
Just adding link to PR so anyone can check on status
```suggestion
  // To be removed after NEXT layer overlay PR is accepted on NEXT branch
  // https://github.com/grommet/grommet-theme-hpe/pull/62
  layer: {
```